packages/terra-menu/src/Menu.jsx

```diff
@@ -53,12 +57,21 @@ const propTypes = {
    * Otherwise, the menu will display without an arrow and right aligned.
    */
   isArrowDisplayed: PropTypes.bool,
+  /**
+   * Allows assigning of root element custom data attribute for easy selecting of document base component.
+   */
+  rootSelector: PropTypes.string,
+  /**
+   * Allows assigning of trigger element custom data attribute for easy selecting of triggering component.
+   */
+  triggerElement: PropTypes.object,
```

Comment:
Why is it a prop as against [abstract modal implementation](https://github.com/cerner/terra-framework/blob/master/packages/terra-abstract-modal/src/AbstractModal.jsx#L118)?

Reply:
This [line](https://github.com/cerner/terra-framework/blob/bed6ba818a71d660ae7416724fbd8a3293ecc0c6/packages/terra-abstract-modal/src/AbstractModal.jsx#L118) returns the menu module instead of the button.

Reply:
Prop removed in 15a749c